doc/sphinx/source/users-guide/locality/onClauses.rst

```diff
@@ -0,0 +1,111 @@
+:title: Users Guide: On-Clauses
+
+.. default-domain:: chpl
+
+On-Clauses
+==========
+
+As mentioned previously, all Chapel programs begin execution as a
+single task running on locale #0.  Tasks can execute on remote locales
```

Comment:
Yes, definitely ignoring sublocales for this first draft.  Even when we include sublocales, I could imagine taking a two-pass approach in which initial sections talk about classic-style locales (network-level only) and on-clauses and then a second pass says "actually, locales can also contain sublocales...  (since I think one of the virtues of a users guide is that it need not be 100% precise at all times since its goal is to be readable.  That said, we should also avoid outright lying...).  I'm also open to rewriting what we have to be more general in the future...

I guess maybe that future isn't so far off.  If we had good numa support this year, that's something I'd want to start calling out more precisely in the user's guide...